@chadlwilson the error I am seeing makes it seem like the analyzer "works" but detects zero dependencies:

```
[ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.938 s <<< FAILURE! -- in org.owasp.dependencycheck.analyzer.YarnAuditAnalyzerIT$Classic
[ERROR] org.owasp.dependencycheck.analyzer.YarnAuditAnalyzerIT.testAnalyzePackageYarnClassicOnYarnBerryLockfile -- Time elapsed: 1.063 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: <org.owasp.dependencycheck.analyzer.exception.AnalysisException> but was: <org.opentest4j.AssertionFailedError>
        at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:67)
        at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
        at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3128)
        at org.owasp.dependencycheck.analyzer.YarnAuditAnalyzerIT$Classic.testAnalyzePackageYarnClassicOnYarnBerryLockfile(YarnAuditAnalyzerIT.java:69)
        at java.base/java.lang.reflect.Method.invoke(Method.java:565)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
Caused by: org.opentest4j.AssertionFailedError: More than 0 dependencies should be identified ==> expected: <true> but was: <false>
        at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at org.junit.jupiter.api.AssertTrue.failNotTrue(AssertTrue.java:63)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:36)
        at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:214)
        at org.owasp.dependencycheck.analyzer.YarnAuditAnalyzerIT.testAnalyzeForMimeDb(YarnAuditAnalyzerIT.java:119)
        at org.owasp.dependencycheck.analyzer.YarnAuditAnalyzerIT$Classic.lambda$testAnalyzePackageYarnClassicOnYarnBerryLockfile$0(YarnAuditAnalyzerIT.java:69)
        at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:53)
        ... 7 more
```

--- a/core/src/test/java/org/owasp/dependencycheck/analyzer/YarnAuditAnalyzerIT.java
+++ b/core/src/test/java/org/owasp/dependencycheck/analyzer/YarnAuditAnalyzerIT.java
@@ -66,7 +66,7 @@ class YarnAuditAnalyzerIT extends BaseTest {
         @Disabled("broken test case on my local machine - needs further investigation")
         @Test
         void testAnalyzePackageYarnClassicOnYarnBerryLockfile() {
-            AnalysisException exception = assertThrows(AnalysisException.class, () -> testAnalyzeForMimeDb("yarn/yarn-classic-audit-bad-berry-lockfile/yarn.lock"));
+            AnalysisException exception = assertThrows(AnalysisException.class, () -> testAnalyzeForUglifyJs("yarn/yarn-classic-audit-bad-berry-lockfile/yarn.lock"));
             assertThat(exception.getMessage(), containsString("No results from Yarn Classic (offline step) - possibly trying to use classic analyzer on Yarn Berry lockfile"));
         }
     }
@@ -111,23 +111,6 @@ class YarnAuditAnalyzerIT extends BaseTest {
             analyzer.analyze(toScan, engine);
             assertEquals(0, engine.getDependencies().length, "No dependency should be identified");
         }
-    }
-
-    private void testAnalyzeForMimeDb(String yarnLockFile) throws Exception {
-        final Dependency toScan = new Dependency(BaseTest.getResourceAsFile(this, yarnLockFile));
-        analyzer.analyze(toScan, engine);
-        assertTrue(0 < engine.getDependencies().length, "More than 0 dependencies should be identified");
-        boolean found = false;
-        for (Dependency result : engine.getDependencies()) {
-            if ("yarn.lock?mime-db".equals(result.getFileName())) {
-                found = true;
-                assertTrue(result.getEvidence(EvidenceType.VENDOR).toString().contains("mime-db"));
-                assertTrue(result.getEvidence(EvidenceType.PRODUCT).toString().contains("mime-db"));
-                assertTrue(result.getEvidence(EvidenceType.VERSION).toString().contains("1.5.4"), "Unable to find version 1.5.4: " + result.getEvidence(EvidenceType.VERSION).toString());
-                assertTrue(result.isVirtual());
-            }
-        }
-        assertTrue(found, "Uglify was not found");
     }
 
     private void testAnalyzeForUglifyJs(String yarnLockFile) throws Exception {

